pyffmpeg/__init__.py

```diff
@@ -225,21 +243,15 @@ def options(self, opts):
         # add ffmpeg
         options = " ".join([self._ffmpeg_file, options])
 
-        self.logger.info(f"Shell: {SHELL}")
+        if self.enable_log:
+            self.logger.info(f"Shell: {SHELL}")
 
         if not SHELL:
             options = shlex.split(options, posix=False)
 
         out = Popen(options, shell=SHELL, stdin=PIPE, stdout=PIPE, stderr=PIPE)
         self._ffmpeg_instances['options'] = out
-        stderr = str(out.stderr.read(), 'utf-8')
-        if stderr and 'Output #0' not in stderr:
-            self.error = stderr.rsplit('\r\n', maxsplit=2)[-2]
-            self.logger.error(self.error)
-            raise Exception(self.error)
-        else:
-            self.error = ''
-            self.logger.info('Conversion Done')
```

Comment:
Standard error, in this case, `stderr`, refers to whatever text was printed to the console whiles the application was still running. It is assumed to have been an error message. Standard output is the text that is printed after an application exits, it is the return statement of the application. But now, from what I see, your code is just putting every standard error text into the error variable. This code checks if there was an "output" information, which is the information FFMpeg prints on the output file, if there wasn't then was an error.